Upgrade dependencies

andersk opened the pull request:
Many of our dependencies (svgicons2svgfont, ttf2woff2, url-join, file-type, read-chunk) are now ESM-only, so we need to convert to ESM as well. This is a breaking change, but it’s the only option going forward.

It’s unclear why this project has both package-lock.json and yarn.lock, but I have updated both. Ideally we should pick one and delete the other; otherwise npm and yarn will fight with each other and repeatedly add useless yarn.lock churn to the Git history.

boris-petrov committed:
Replace `mkdirp` with native `fs` calls

andersk committed:
Convert to ES modules

Many of our dependencies (svgicons2svgfont, ttf2woff2, url-join,
file-type, read-chunk) are now ESM-only, so we need to convert to ESM
as well.  This is a breaking change, but it’s the only option going
forward.

Signed-off-by: Anders Kaseorg <andersk@mit.edu>

andersk committed:
Remove deprecated q

Signed-off-by: Anders Kaseorg <andersk@mit.edu>

andersk committed:
Replace deprecated node-sass with sass

Signed-off-by: Anders Kaseorg <andersk@mit.edu>

andersk committed:
Remove unnecessary dependency url-join

Signed-off-by: Anders Kaseorg <andersk@mit.edu>

andersk committed:
Remove unnecessary dependency read-chunk

Signed-off-by: Anders Kaseorg <andersk@mit.edu>

thiagocoelho commented:
Hello there!

I'm using this awesome package [webfonts-loader](https://github.com/jeerbl/webfonts-loader/blob/v8.0.1/package.json) which depends on https://github.com/vusion
[webfonts-generator](https://github.com/vusion/webfonts-generator)

I've encountered an issue with `svgicons2svgfont`, where icons are rendered with incorrect dimensions.

I noticed that this PR seems to fix this problem. Could you please let me know if this fix will be officially released soon?

Thanks in advance :)

andersk committed:
Upgrade dependencies

It’s unclear why this project has both package-lock.json and
yarn.lock, but I have updated both.

Signed-off-by: Anders Kaseorg <andersk@mit.edu>

mg-aceik commented:
@andersk since this fork seems abandoned can you push your changes as a new npm package?
